common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -109,6 +109,8 @@ public enum Feature {
         HOSTNAME_V1("Hostname Options V1", Type.DEFAULT),
         //HOSTNAME_V2("Hostname Options V2", Type.DEFAULT, 2),
 
+        OID4VC_VCI("Support for the OID4VCI protocol as part of OID4VC.", Type.PREVIEW),
```

Comment:
Is it possible to make the feature `EXPERIMENTAL` instead of `PREVIEW` ? For the inspiration, you can take a look at this PR, which introduces some experimental feature what are all the various things needed to make tests working https://github.com/keycloak/keycloak/pull/26389 .